pangeo_forge_runner/commands/bake.py

```diff
@@ -55,6 +55,16 @@ class Bake(BaseCommand):
         """,
     )
 
+    recipe_id = Unicode(
+        default_value="",
```

Comment:
Let's set the default to `None` insteadd?

Reply:
Yes that's better. Will fix now.

Reply:
fixed in [ba13d4d](https://github.com/yuvipanda/pangeo-forge-runner/pull/14/commits/ba13d4d60f7d4d37aaea80660456979940ffa1bd)